src/CommunityToolkit.Maui.Core/Handlers/DrawingView/DrawingViewHandler.shared.cs

```diff
@@ -41,7 +41,7 @@ public DrawingViewHandler() : base(DrawingViewMapper)
 	}
 }
 
-#if ANDROID || IOS || MACCATALYST || WINDOWS
+#if __MOBILE__ || WINDOWS
```

Comment:
Is Tizen considered `__MOBILE__`?

Reply:
probably yes.
it is definitely includes MacCatalyst. I reverted it to make sure we add tizen support explicitly